exporting/README.md

```diff
@@ -123,53 +99,69 @@ In `/etc/netdata/exporting.conf` you should have something like this:
     update every = 10
 
 [prometheus:exporter]
-    send charts matching = system.processes
+    send names instead of ids = yes
+    send configured labels = yes
+    end automatic labels = no
+    send charts matching = *
+    send hosts matching = localhost *
+    prefix = netdata
 
-[graphite:my_instance_1]
+[graphite:my_graphite_instance]
     enabled = yes
     destination = localhost:2003
-    data source = average
-    prefix = Netdata
-    hostname = my-name
-    update every = 10
-    buffer on failures = 10
-    timeout ms = 20000
-    send charts matching = *
-    send hosts matching = localhost *
-    send names instead of ids = yes
```

Comment:
Wouldn't it be better to leave all possible options at least for one instance?

Reply:
Good point. I added them back in to the Graphite block and explained in the paragraph above that they can be used as a reference for how more options looks/operates.